README.md

```diff
@@ -172,19 +172,27 @@ This method is invoked when an error occurred during the use of the Drop-in. Dis
 
 ---
 
+```swift
+func didComplete(from component: DropInComponent)
+```
+
+This method is invoked when the action component finishes, without any further steps needed by the application, for example in case of voucher payment methods. The application just needs to dismiss the `DropInComponent`..
+
+---
+
 ```swift
 func didCancel(component: PaymentComponent, from dropInComponent: DropInComponent)
 ```
 
-This method is invoked when user closes a payment component managed by Drop-in.
+This optional method is invoked when user closes a payment component managed by Drop-in.
 
 ---
 
 ```swift
-func didComplete(from component: DropInComponent)
+func didOpenExternalApplication(_ component: DropInComponent)
 ```
 
-This method is invoked when the action component finishes, without any further steps needed by the application, for example in case of voucher payment methods. The application just needs to dismiss the `DropInComponent`..
+This optional method is invoked when redirect to external application occurred.
```

Comment:
maybe `This optional method is invoked after a redirect to an external application has occurred.`

Reply:
👍